app/mailers/ticket_mailer.rb

```diff
@@ -124,7 +124,8 @@ def receive(email)
         message_id: email.message_id,
         content_type: content_type,
         to_email_address: to_email_address,
-        raw_message: StringIO.new(email.to_s)
+        raw_message: StringIO.new(email.to_s),
+        unread_users: User.all # none of the users could have read this
```

Comment:
No forget I asked, I think this should be done together with a change in the other file. I'll try to come up with a better solution today.